app/user/__init__.py

```diff
@@ -25,21 +43,3 @@ def load_user(id):
     # Set up sanity checks.
     from . import sanity
     getattr(app, 'sanity_check_modules', []).append(sanity)
-
```

Comment:
Is this just code movement? If yes, why, and can we put that in a separate commit?